### ISSUE TITLE
Dead plugin? Hasn't worked for months...

### ISSUE THREAD

**Issue by VirgileVILE** [Lacro59/playnite-successstory-plugin#349]
-causes an error when launching playnite after starting the computer...
-does infinite loading on the SuccessStory page...
-slows down the pages of the games on which it must display achievements...
Conclusion: no longer of any interest, I'm going to delete it since it doesn't work properly.

**Comment by Lacro59** [comment]
Can you post your playnite.log & extension.log?

**Comment by VirgileVILE** [comment]
If I find these files, I will do so. ><

**Comment by Lacro59** [comment]
In "%appdata%\Playnite" (if installed version) or "Playnite" folder (if portable version).

**Comment by darklinkpower** [comment]
@Lacro59 I highly recommend to ignore this person and maybe block them. They are very rude, entitled and were not taught manners by their parents, see how they behaved in the Playnite repo: https://github.com/JosefNemec/Playnite/issues/3045 or https://github.com/JosefNemec/PlayniteExtensions/issues/219

All their GitHub interactions have no manners so just giving this advice.

**Comment by VirgileVILE** [comment]
That's it, everything is there.
https://www.mediafire.com/file/4b7xysz43qxeaoi/logs.zip/file

**Comment by VirgileVILE** [comment]
@darklinkpower I used to like your work, but really, and now you've given me a reason to delete all your extensions. Thanks for the judgement.

**Comment by Jeshibu** [comment]
> @darklinkpower I liked your work, but really, and now you're giving me a reason to delete all your extensions without hesitation. If it's true that I'm quite direct, it's because English is not my first language and I don't want to get into complicated discussions. On the other hand, the Playnite developer talked to me like an asshole and since that day I don't respect that guy anymore, so I put him in his place. So you can insult me all you want, I don't care, but don't you dare bring my parents into this, they have nothing to do with the behaviour of their 40 year old autistic son.

Crow was not rude to you, he was blunt/direct at worst. You flew into a rage for no reason. I don't agree with trying to blacklist you from other projects, but you were not the one who was wronged first. Do some introspection as to your manners.

**Comment by VirgileVILE** [comment]
All he had to do was talk to me like that, I'm followed by doctors because I'm very sensitive and know no limits to ask for redress, even if it means pestering the person on 150 different sites as long as there are accounts to communicate. But I agree, he couldn't have guessed it, it's true...

**Comment by Jeshibu** [comment]
> All he had to do was talk to me like that, I'm followed by doctors because I'm very sensitive and know no limits to ask for redress, even if it means pestering the person on 150 different sites as long as there are accounts to communicate. But I agree, he couldn't have guessed it, it's true...

Nobody online can look into a crystal ball to see that you're extra sensitive. That sounds like a personal issue you should work on, not something you should demand everyone else in the world change. If you're chasing people across dozens of sites to demand an apology, that's something you should talk to your doctors about too. Especially at 40 years old, you should know better.

**Comment by VirgileVILE** [comment]
No, since I was a child I've had to adapt to a world that doesn't fit me, and I've been persecuted, if it were up to me I would reformat the human race without any hesitation. As long as people, by themselves, don't show a minimum of respect, I wouldn't either!

**Comment by Jeshibu** [comment]
> No, since I was a child I've had to adapt to a world that doesn't fit me, and I've been persecuted, if it were up to me I would reformat the human race without any hesitation. As long as people, by themselves, don't show a minimum of respect, I wouldn't either!

You were shown as much respect as anyone else. You were offered help and you took offense at some imagined rudeness.

**Comment by VirgileVILE** [comment]
Well, look, all I'm asking is to be able to have normal relationships with people one day. And my doctors encourage me to take the first step even if it costs me... so, ok, I'll do it: Can we, with the developer, talk about it calmly, and find a solution that doesn't leave a bitter taste to any of us?

**Comment by Jeshibu** [comment]
> Well, look, all I'm asking is to be able to have normal relationships with people one day. And my doctors encourage me to take the first step even if it costs me... so, ok, I'll do it: Can we, with the developer, talk about it calmly, and find a solution that doesn't leave a bitter taste to any of us?

Well, all your attempts at communication have been angry rants, which isn't going to win you any friends. There's not much to talk about if you demand an apology after you've wronged someone.

**Comment by VirgileVILE** [comment]
From my point of view, I was the one who was offended in the first place, I am willing to consider, however complicated it may be for my pathology, that I was wrong. But I won't be able to move on if I can't talk to these people.

**Comment by VirgileVILE** [comment]
(Either way the damage is done, you'll never get the idea that this guy is a bastard who banishes people rather than trying to argue, the kind of jerk that today's society has produced and that puts me off to no end. arrived today i will learn how to develop my own game application manager, so much now i hate this guy and his team and it boosts my motivation to humiliate them by doing better in France.)

**Comment by Jeshibu** [comment]
> (Either way the damage is done, you'll never get the idea that this guy is a bastard who banishes people rather than trying to argue, the kind of jerk that today's society has produced and that puts me off to no end. arrived today i will learn how to develop my own game application manager, so much now i hate this guy and his team and it boosts my motivation to humiliate them by doing better in France.)

Good luck!

**Comment by LiterallyJohnny** [comment]
Well that was interesting.
